app/workers/publishing_api_worker.rb

```diff
@@ -39,4 +44,13 @@ def save_draft_of_unpublished_edition(unpublishing)
       Whitehall::PublishingApi.save_draft_async(draft)
     end
   end
+
+  def handle_error(error)
+    if error.code >= 500
+      raise error
+    else
+      explanation = "The message is a duplicate and does not need to be retried"
+      Airbrake.notify_or_ignore(error, parameters: { explanation: explanation })
```

Comment:
I'm not sure I understand what this is trying to achieve. This will still post the error to Errbit with additional annotation, so I don't think this will reduce noise? From http://www.rubydoc.info/github/thoughtbot/airbrake/Airbrake.notify, `parameters` doesn't appear to be a valid option?